lib/json_schemer/schema/base.rb

```diff
@@ -29,29 +30,39 @@ def merge(
         :eol => '\z'
       }.freeze
 
+      INSERT_DEFAULT_PROPERTY = proc do |data, property, property_schema|
```

Comment:
Does this need to include the last argument (`_parent`)?

Reply:
It does not have to, since it's not a `lambda`, but it probably should for consistency. I'll change it.